app/javascript/components/components/Header.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
+import logo from '/Users/sahtra/Desktop/Projects/apartment_app/app/javascript/components/assets/housu-icon-black.png'
```

Comment:
This logo should be brought into your app. It looks like it is coming from your root directory?